src/core/Meganav/data.tsx

```diff
@@ -217,4 +217,5 @@ export const productsForNav = {
     ...products.assetTracking,
     link: "/solutions/asset-tracking",
   },
+  liveObjects: { ...products.liveObjects, link: "/docs/liveobjects" },
```

Comment:
we dont need to update the Meganav yet since in the ProductTile the url is already `docs/liveobjects` , we can only update it if we have the product LO page already.

Reply:
Ahh good point! I was in auto-pilot and forgot they point to docs by default.